codeSnippets/snippets/tutorial-full-stack-task-manager/shared/src/commonMain/kotlin/Greeting.kt

```diff
@@ -0,0 +1,7 @@
+class Greeting {
```

Comment:
Is this file used?

Reply:
Not in the final application but we explain its function in the [Examine the project](https://ktor.shared.aws.intellij.net/docs/full-stack-development-with-kotlin-multiplatform.html#examine-project) section.